src/views/outreach/host/home.jsx

```diff
@@ -1,44 +1,58 @@
 import React from 'react';
 
-const HostHomeSection = () => (
-    <div>
-        <div className="content-section">
-            <div className="content-section-description">
-                <p>{'The '}
-                    <a
-                        href="http://ase.tufts.edu/devtech/index.html"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        DevTech Research Group
-                    </a>{ ' at ' }
-                    <a
-                        href="https://www.tufts.edu/"
-                        target="_blank"
-                        rel="noopener noreferrer"
-                    >
-                        Tufts University
-                    </a>,
-                co-creator of ScratchJr, is conducting a study about how families
-                learn programming together. We would like to invite you to participate in
-                the research portion of ScratchJr
-                Family Days, though it is not
-                mandatory for running a ScratchJr family event.
-                Should you wish to participate, please sign up by filling out a
-                short questionnaire below. We appreciate your feedback!</p>
+class HostHomeSection extends React.Component {
+    componentDidMount () {
+        const ifr = this.ifr;
+        ifr.addEventListener('load',
+            () => {
+                window.scrollTo(0, 0);
+            });
+    }
```

Comment:
I think you could simplify this a little bit and avoid using an anonymous function as a ref at the same time. Rename `componentDidMount` to `scrollIframeOnLoad (ifr)` and then assign `this.scrollIframeOnLoad` as the ref for the iframe. It doesn't matter here but in general it's bad for performance to use anonymous/arrow functions as refs because then you're creating functions in `render`, rather than just giving references to functions.